localization/english/wc_regiment_l_english.yml

```diff
@@ -14,7 +14,155 @@
+ sentinel_flavor:0 "#F Being the main force of the Kaldorei, Sentinels adopt flexible strategies in combat, particularly favoring ambushes or hit-and-run tactics."
+ siren_flavor:0 "#F The Naga are descended from the Highborne, and their ancestry is particularly clear with Naga Sirens, whose mastery of magic is undeniable."
+ snow_hunter_flavor:0 "#F Ours is a hard land, fraught with peril. The bravest of our people, the snow hunters, embrace the perils of winter, and are stronger for it."
+ swarm_flavor:0 "#F The remnants of the Aqir are an undeniable threat, though they differ from most races in favoring quantity over quality in their soldiers, shown through their swarming tactics."
+ thornguard_flavor:0 "#F Quilboar are well known for their ferocity in battle. The elite Thornguard take this to an entirely new level, tearing through opposition like a scythe through wheat."
+ tidehunter_flavor:0 "#F Aaaaaughibbrgubugbugrguburgle! Mrgllll glrrm gl!"
+ ursa_flavor:0 "#F The indomitable Ursas pounce with ferocious fury on all intruders who threaten their domain and nature itself."
+ warbrave_flavor:0 "#F The bravest of the Tauren have passed many tests of body and spirit and are at the forefront of preserving the legacy of the Ancestors and the Earth Mother."
+ war_dinosaur_flavor:0 "#F Being a magnificent relic of a forgotten age, none would question the power of a dinosaur. Truly, only our proud species is worthy of such grand steeds."
+ wolfrider_flavor:0 "#F Orcs and wolves share an unbreakable bond. Individually strong, but together, unstoppable. Skilled, cunning, and fast, wolfriders are among the most deadly enemies one could encounter."
+ wyvern_rider_flavor:0 "#F Together with our new allies, the wyverns, we have created a unit of skilled aerial combatants, the Wind Riders. At long last, the skies are within our grasp."
+
+ ##Early Medival
+ abomination_flavor:0 "#F Our creation of the abomination has provided us with an impossibly powerful and durable soldier. Lesser minds might reject them on grounds of ‘morals’, but we know their value."
+ baradian_marine_flavor:0 "#F The elite of our military, Baradian Marines know no fear, only glorious victory. In peacetime they guard the prison of Tol Barad, but it is in war that they truly shine."
+ bearpaws_flavor:0 "#F"
+ blackrock_legionnaire_flavor:0 "#F The best of the Blackrock Clan, the Legionnaires defy expectations as an orcish force with iron discipline. They conquer all in their path, showing no mercy."
+ chaosborn_flavor:0 "#F"
+ chimaera_flavor:0 "#F The fearsome two-headed Chimaeras circle the dark forests of northern Kalimdor, fighting with their terrible breath against all who desecrate the purity of their homeland."
+ cloud_serpents_flavor:0 "#F Long have we held close bonds with our friends in the sky. Now, our bond is closer than ever, as we ride the Cloud Serpents to new heights."
+ faerie_dragon_flavor:0 "#F"
```

Comment:
Bearpaws, Chaosborn, and Faerie Dragons missing.

Reply:
Added.